### DISCUSSION TITLE
I am unable to access the host configuration pages via external network

### DISCUSSION THREAD

**Discussion by dsanttos12** [ClassicOldSong/Apollo#116] [General]
I am unable to access the Apollo settings via browser, even though I have configured the host settings to be accessible on an external network. I also redirected port 47990, but even after entering the login and password correctly, the settings page does not load as shown in the image.
![20241122_140324](https://github.com/user-attachments/assets/8d12cdd8-ba51-4906-90e1-1f964ea42600)

**Comment by ClassicOldSong** [comment]
Could you open the dev tools and see what error message it says?

**Reply by dsanttos12** [reply]
Where?

**Reply by ClassicOldSong** [reply]
Browser dev tools

Right click on the page and select "Inspect Element"

**Reply by ClassicOldSong** [reply]
How are you forwarding the page? Do you redirect the port directly or used a reverse proxy like nginx and put the login page in another path?

**Reply by dsanttos12** [reply]
I redirected all ports directly mainly port 47990

**Reply by ClassicOldSong** [reply]
You'll need to check the dev tools then

**Reply by dsanttos12** [reply]
I managed to solve the problem

**Reply by ClassicOldSong** [reply]
Glad to know, so what's the problem?

**Reply by dsanttos12** [reply]
I just had to give the space in the user section, and I had forgotten about it
